app/views/ers_main.scala.html

```diff
@@ -58,7 +60,7 @@
     headBlock = Some(hmrcHead(Some(head))),
     headerBlock = Some(hmrcStandardHeader(
         serviceName = Some(messages("service.name")),
-        signOutUrl = None,
+        signOutUrl = Some(applicationConfig.signOut),
```

Comment:
this is in DDCE-1912, would be good to link the tickets

Reply:
(or leave it for then, your call)

Reply:
Good spot, will do